### ISSUE TITLE
Windows install instruction missing

### ISSUE THREAD

**Issue by hrump** [Yubico/libfido2#516]
Is Windows this still supported? The referenced .actions/windows directory doesn't appear to be present

**Comment by James-ZHANG** [comment]
There are build scripts under the top /windows directory.

**Comment by hrump** [comment]
Are there any additional instructions on which scripts to run or what configuration is needed? I'm getting a number of errors about the x64 platform specification not being supported and some COMPILER definitions not being set.

**Comment by martelletto** [comment]
@nemchik, you will need a version of ssh built with FIDO2 support, such as the latest release of https://github.com/PowerShell/Win32-OpenSSH/. You can try to follow the instructions at https://upyesp.org/posts/winwdows-ssh-mfa-fido2-yubikey/ (disclaimer: not official Yubico documentation).

**Comment by nemchik** [comment]
> @nemchik, you will need a version of ssh built with FIDO2 support, such as the latest release of https://github.com/PowerShell/Win32-OpenSSH/. You can try to follow the instructions at https://upyesp.org/posts/winwdows-ssh-mfa-fido2-yubikey/ (disclaimer: not official Yubico documentation).

Thanks! I found these instructions https://github.com/PowerShell/Win32-OpenSSH/wiki/FIDO---U2F-usage to be very helpful.

In my adventures I ended up running `ssh-keygen-O "resident" ...` multiple times and it seems my yubikey is now storing multiple key pairs (evident when running `ssh-keygen -K`, it outputs multiple key pairs to my disk). Is there a way to revoke/remove/delete these (hopefully without resetting the key completely)? I opened support request 381633 via https://support.yubico.com/hc/en-us/requests/new asking this question as well.
